src/components/productoverview/ProductSorting.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <select data-test="sorting-select"
+            name="sort"
+            id="sort"
+            v-model="sortBy">
+        <option value="">--</option>
+        <option value="newest">Newest</option>
```

Comment:
Usually, our templates never have anything written directly. Their internationalization should be flexible depending on the selected language.